README.md

```diff
@@ -18,13 +18,13 @@ Include the library by adding an `implementation` reference to `dependencies` bl
 For [Java](https://mvnrepository.com/artifact/io.ably/ably-java/latest):
 
 ```groovy
-implementation 'io.ably:ably-java:2.0'
+implementation 'io.ably:ably-java:1.2.18'
```

Comment:
I think it should stay as `2.0` 😉

Reply:
Yes, it will be 2.0.0 per semantics versioning but we can do that before merging v2 into the main.

Reply:
Gradle files may have slipped in from merging main. I have removed it https://github.com/ably/ably-java/pull/849/commits/d9c876b4e64150e1150b3a4e78b62d3cc56ee27a